source/mongorestore.txt

```diff
@@ -551,11 +551,13 @@ Options
 .. option:: --oplogReplay
 
    After restoring the database dump, replays the :term:`oplog` entries
-   from a bson file.
+   from an ``oplog.bson`` file.
    When used in conjunction with :option:`mongodump --oplog`,
    :binary:`mongorestore --oplogReplay <~bin.mongorestore>`
-   restores the database to the point-in-time backup captured with the
-   ``mongodump --oplog`` command. 
+   restores the database to the time the ``oplog.bson`` file was created
```

Comment:
Rather than framing this as restoring to a point in time, I suggest using phrasing like my comment above:
"To apply those oplog entries during the restore, use mongorestore --oplogReplay. Using mongodump --oplog in conjunction with mongorestore --oplogReplay ensures the data is up-to-date with all writes that occurred during the dump. However, if any writes occur after mongodump completes, those will not be captured in that dump."